webui/src/pages/repositories/repository/commits/commit/index.tsx

```diff
@@ -104,18 +110,25 @@ const CommitMetadataTable = ({ commit }) => {
                 </tr>
             </thead>
             <tbody>
-            {keys.map(key => (
-                <tr key={key}>
-                    <td><code>{key}</code></td>
-                    <td><code>{commit.metadata[key]}</code></td>
-                </tr>
-            ))}
+                {keys.map(key =>
+                    <MetadataRow metadata_key={key} metadata_value={commit.metadata[key]}/>)}
             </tbody>
         </Table>
         </>
     );
 };
 
+const CommitMetadataUIButtons = ({ commit }) => {
+    const keys = getKeysOrNull(commit.metadata);
+    if (!keys) return null;
+
+    return (
```

Comment:
Thanks!  I always get confused by the fact that user-defined components can have keys with no further effort.  I used [the old docs](https://legacy.reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys) as a guide, because the new docs don't seem to give a proper example of keys in user-defined components.  So... am I correct that my component MetadataUIButton does nothing with its key, and everything "just works"?